_includes/head.html

```diff
@@ -15,5 +15,28 @@
   </style>
   <link rel="manifest" href="{{ site.url }}/manifest.json">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ site.url }}/feed.xml">
-  <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
-</head>
+
+  {% if site.title %}
+    <title>{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}</title>
+  {% endif %}
+
+  {% if page.layout == 'permalink' %}
+    <meta name="twitter:card" content="summary">
+  {% else %}
+    <meta name="twitter:card" content="summary_large_image">
+  {% endif %}
+
+  {% if site.title %}
+    <meta name="twitter:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}">
```

Comment:
It seems `og:title` was omitted entirely?

According to Twitter's docs, you should _only_ need `og:title`, but if you're seeing different results, you can combine the two like so:

```
<meta property="og:title" name="twitter:title" content="{% if page.title %}{{ page.title }} - {% endif %}{{ site.title }}">
```